dashboard/app/controllers/api_controller.rb

```diff
@@ -65,12 +65,14 @@ def import_clever_classroom
     begin
       yield service
 
-      if client.access_token != current_user.oauth_token
-        current_user.update!(
-          oauth_token: client.access_token,
-          oauth_token_expiration: client.expires_in + Time.now.to_i,
-        )
-      end
+      current_user.update_oauth_credential_tokens(
+        provider: AuthenticationOption::GOOGLE,
+        credentials: {
+          token: client.access_token,
+          refresh_token: client.refresh_token,
+          expires_at: client.expires_at + Time.now.to_i,
```

Comment:
Looking at [this Honeybadger error](), it appears that we sometimes receive a `nil` value for `client.expires_at`. In that case, what do we want to do? Set `expires_at` to `Time.now.to_i`?

Reply:
I don't actually know what `expires_at` gets used for - it looks we pass it into the Signet client, above.  Maybe we should just store `nil` if that's what we get back from the service?  This is probably worth finding some documentation about...